book/tables/ADA/adat01.qmd

```diff
@@ -7,7 +7,7 @@ subtitle: Baseline Prevalence and Incidence of Treatment Emergent ADA
 
 {{< include ../../_utils/envir_hook.qmd >}}
 
-:::: {.panel-tabset}
+::: panel-tabset
```

Comment:
why these and other happened? I will revert if it is only my local

Reply:
yes - please do review your changes carefully
in case of webr - break line matters - this is unfortunately very sensitive implementation on that regard

Reply:
I knew that, but with Rstudio, this is impossible to change. Every time I save, it automatically reverts to the above! Using GitHub. Have you encountered this before?